src/main/scala/com/databricks/labs/overwatch/pipeline/DbsqlTransforms.scala

```diff
@@ -0,0 +1,130 @@
+package com.databricks.labs.overwatch.pipeline
+
+import com.databricks.labs.overwatch.pipeline.PipelineFunctions.fillForward
+import com.databricks.labs.overwatch.pipeline.TransformFunctions._
+import com.databricks.labs.overwatch.utils.SchemaTools.structFromJson
+import com.databricks.labs.overwatch.utils._
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.sql.expressions.{Window, WindowSpec}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{Column, DataFrame}
+
+object DbsqlTransforms extends SparkSessionWrapper {
+
+  import spark.implicits._
+  private val logger: Logger = Logger.getLogger(this.getClass)
+  val responseSuccessFilter: Column = $"response.statusCode" === 200
+
+  /**
+   * BEGIN DBSQL generic functions
+   */
+
+  private val auditBaseCols: Array[Column] = Array(
+    'timestamp, 'date, 'organization_id, 'serviceName, 'actionName,
+    $"userIdentity.email".alias("userEmail"), 'requestId, 'response)
+
+  def deriveWarehouseId(): Column = {
+    when(('actionName === "createEndpoint" || 'actionName === "createWarehouse")
+      && responseSuccessFilter,
```

Comment:
should the response filter go here? I think it should go on the first reference of auditRawDF. Seems like the logic is wrong to put the responseSuccessFulter here.

Reply:
done. Removed from here and added at auditRawDf